templates/firefox-ios.xliff

```diff
@@ -2270,7 +2414,7 @@
     </header>
     <body>
       <trans-unit id="CFBundleDisplayName">
-        <source>Share To</source>
+        <source>$(MOZ_BUNDLE_DISPLAY_NAME)</source>
```

Comment:
Yes, the Share To extension should be named 'Firefox' or 'Firefox Beta' - it should have the same name as on the Home Screen - so not translated.